index.js

```diff
@@ -1,88 +1,121 @@
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
+const name = 'Matt'
 
-// console.log(name)
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
+function drinkWater(thirstLevel){
+  console.log('Man I sure am thirsty');
+  thirstLevel--;
+  console.log('Ahh that hits the spot');
+  return thirstLevel
+}
+
+console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num, maybeNum){
+  if (typeof maybeNum === 'number' && num === maybeNum){
+    return "same same"
+  } else if (num == maybeNum) {
+    return "same same (but different)";
+  } else { return "different"}
 
-// console.log(sameSameButDifferent(5, 5)) 
+}
+
+
+console.log(sameSameButDifferent(5, 5)) 
 // // => "same same"
 
-// console.log(sameSameButDifferent(123, "123")) 
+console.log(sameSameButDifferent(123, "123")) 
 // // => "same same (but different)"
 
-// console.log(sameSameButDifferent(5, 7)) 
+console.log(sameSameButDifferent(5, 7)) 
 // // => "different"
 
-// console.log(sameSameButDifferent(123, "122")) 
+console.log(sameSameButDifferent(123, "122")) 
 // // => "different"
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 5 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
+
+function updateGrade(student, grade){
+  student.grade = grade;
+}
+
+
+console.log("%cQuestion 5", "color: red")
+const student1 = { name: "Duane", grade: 88 }
+updateGrade(student1, 92)
+console.log(student1)
 // // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red")
+console.log("%c----------", "color: red")
 
 
 
 // ***** Question 6 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
+console.log("%cQuestion 6", "color: red")
+
+function printNameAndPhones(users){
+  users.forEach( el => console.log(el.name,'\n',"Cell: "+el.phones.cell,'\n',"Office: "+el.phones.office))
+}
```

Comment:
Nice! Using a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) may look cleaner in this case since there's no need for the newline character or the addition operator. Example:

```suggestion
function printNameAndPhones(users){
  users.forEach( el => {
  console.log(`${el.name}
  Cell: ${el.phones.cell}
  Office: ${el.phones.office}`)
    )
  }
}
```